source/dotnet/Library/AdaptiveCards/AdaptiveCard.cs

```diff
@@ -257,6 +258,36 @@ public bool ShouldSerializeJsonSchema()
         [DefaultValue(null)]
         public AdaptiveAuthentication Authentication { get; set; }
 
+		/// <summary>
+        /// Defines originator id for card.
+        /// </summary>
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+#if !NETSTANDARD1_3
+        [XmlElement]
+#endif
+        [DefaultValue(null)]
+        public string Originator { get; set; }
```

Comment:
do you have any examples about what this Originator should look like? what is this value used for?